src/nomad_material_processing/__init__.py

```diff
@@ -23,33 +23,189 @@
 
 
-class Substrate(CompositeSystem):
+class Geometry(ArchiveSection):
     '''
-    A thin free standing sheet of material. Not to be confused with the substrate role
-    during a deposition, which can be a `Substrate` with `ThinFilm`(s) on it.
+    Geometrical shape attributes of a system.
+    The children of this class represent concrete geometrical shapes.
     '''
-    m_def = Section()
-    thickness = Quantity(
+    m_def = Section(
+    )
+    volume = Quantity(
         type=float,
-        description='''
-        The (average) thickness of the substrate.
-        ''',
+        description='docs',
+        a_eln={
+            "component": "NumberEditQuantity"
+        },
+        unit="meter ** 3",
+    )
+
+
+class Parallelepiped(Geometry):
+    '''
+    Six-faced polyhedron with each pair of opposite faces parallel and equal in size, 
+    characterized by rectangular sides and parallelogram faces.
+    '''
+    m_def = Section(
+    )
+    height = Quantity(
+        type=float,
+        description='dimension z',
+        a_eln={
+            "component": "NumberEditQuantity",
+            "defaultDisplayUnit": "nanometer",
+            "label": "Height (z)",
+        },
+        unit="meter",
+    )
+    width = Quantity(
+        type=float,
+        description='dimension x',
         a_eln={
             "component": "NumberEditQuantity",
-            "defaultDisplayUnit": "millimeter"
+            "defaultDisplayUnit": "millimeter",
+            "label": "Width (x)"
         },
         unit="meter",
     )
+    length = Quantity(
+        type=float,
+        description='dimension y',
+        a_eln={
+            "component": "NumberEditQuantity",
+            "defaultDisplayUnit": "millimeter",
+            "label": "Length (y)"
+        },
+        unit="meter"
+    )
+    surface_area = Quantity(
+        type=float,
+        description='product of length and width, representing the total exposed area of the primary surface',
+        a_eln={
+            "component": "NumberEditQuantity",
+            "defaultDisplayUnit": "millimeter ** 2",
+            "label": "Surface Area (x*y)"
+        },
+        unit="meter ** 2",
+    )
 
+
+class Miscut(ArchiveSection):
+    '''
+    The miscut in a crystalline substrate refers to 
+    the intentional deviation from a specific crystallographic orientation, 
+    commonly expressed as the angular displacement of a crystal plane.
+    '''
+    angle = Quantity(
+        type=float,
+        description='angular displacement from crystallographic orientation of the substrate',
+        a_eln={
+            "component": "NumberEditQuantity",
+            "defaultDisplayUnit": "deg"
+        },
+        unit="deg",
+    )    
+    angle_deviation = Quantity(
+        type=float,
+        description='uncertainty on the angular displacement',
+        a_eln={
+            "component": "NumberEditQuantity",
+            "defaultDisplayUnit": "deg"
+        },
```

Comment:
Here i definitely think we need a label. Not sure if the "±" sign will work...
```suggestion
        a_eln=ELNAnnotation(
            component=ELNComponentEnum.NumberEditQuantity,
            defaultDisplayUnit='deg',
            label='± Miscut Angle Deviation',
        ),
```